sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -21,6 +24,13 @@ def test_get_users_options_function_should_return_only_users_options(self):
             CustomUser.UserType.EMPLOYEE.name: None,
             CustomUser.UserType.MANAGER.name: 2,
             self.superuser: False,
+            ProjectType.SUSPENDED.name: None,
+            ProjectType.ACTIVE.name: None,
+            ProjectType.COMPLETED.name: None,
+            DataSize.SMALL.name: False,
+            DataSize.MEDIUM.name: False,
+            DataSize.LARGE.name: False,
+            DataSize.EXTRA_LARGE.name: False,
```

Comment:
Okay, so now you have what looks like a ton of options unrelated to your tests, but must be present. This is a perfect moment to move it to the **setUp**. Either declare **mock_options** there and alter necessary values individually in tests or put these values in a separate dicts and concatenate it  with mock_options.